orders/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Register your model in the admin.

Reply:
https://github.com/jino-d/warehouse/pull/1/commits/59ace3fed4a512764e82158d1ee809b3ee45eeb7